README.md

```diff
@@ -66,13 +83,26 @@ Add the following to your `Package.swift` file:
 
 ```swift
 dependencies: [
-    .package(url: "https://github.com/toon-format/toon-swift.git", from: "0.2.0")
+    .package(url: "https://github.com/toon-format/toon-swift.git", from: "0.3.0")
 ]
 ```
 
+Then add the dependency to your target:
+
+```swift
+// For both encoder and decoder:
+.target(name: "YourTarget", dependencies: ["ToonFormat"])
+
+// Or individually:
+.target(name: "YourTarget", dependencies: ["TOONEncoder"])
+.target(name: "YourTarget", dependencies: ["TOONDecoder"])
+```
+
 ## Usage
 
-### Quick Start
+### Encoding
```

Comment:
This is definitely a valid way to structure the discussion. But to keep the heading levels relatively flat, I think we can introduce decoding as a sibling to other operations at the end. Again I think the vast majority of API consumers won't touch `TOONDecoder`, so we can treat it more like an advanced feature than a peer of `TOONEncoder`.

Instead, what if we reverted the heading levels, updated the first basic usage to show round-trip decoding, and then kept just the part about configuring decoding limits?